src/Nri/Ui/ClickableSvg/V1.elm

```diff
@@ -215,16 +216,59 @@ css styles =
 -- TOOLTIPS
 
 
+type alias TooltipSettings msg =
+    { position : Tooltip.Position
+    , isOpen : Bool
+    , onShow : Bool -> msg
+    , id : String
+    }
+
+
+showTooltip : String -> Maybe (TooltipSettings msg) -> Html msg -> Html msg
+showTooltip label maybeSettings buttonOrLink =
+    case maybeSettings of
+        Just { position, onShow, isOpen, id } ->
+            let
+                tooltipSettings =
+                    { trigger = Tooltip.OnHover
+                    , onTrigger = onShow
+                    , isOpen = isOpen
+                    , triggerHtml = buttonOrLink
+                    , extraButtonAttrs = []
+                    , id = id ++ "__clickable-svg-tooltip"
+                    }
+            in
+            Tooltip.tooltip [ Html.text label ]
+                |> Tooltip.withPosition position
+                |> Tooltip.withWidth Tooltip.FitToContent
+                |> Tooltip.withPadding Tooltip.SmallPadding
+                |> Tooltip.primaryLabel tooltipSettings
+
+        Nothing ->
+            buttonOrLink
+
+
+withTooltip : Tooltip.Position -> { id : String, isOpen : Bool, onShow : Bool -> msg } -> Attribute msg
+withTooltip position { id, isOpen, onShow } =
+    set
+        (\config ->
+            { config
+                | tooltip =
+                    Just { position = position, id = id, isOpen = isOpen, onShow = onShow }
+            }
+        )
+
+
 {-| -}
-withTooltipAbove : Attribute msg
+withTooltipAbove : { id : String, isOpen : Bool, onShow : Bool -> msg } -> Attribute msg
 withTooltipAbove =
-    set identity
+    withTooltip Tooltip.OnTop
 
 
 {-| -}
-withTooltipBelow : Attribute msg
+withTooltipBelow : { id : String, isOpen : Bool, onShow : Bool -> msg } -> Attribute msg
 withTooltipBelow =
-    set identity
+    withTooltip Tooltip.OnBottom
```

Comment:
I think it'd make more sense to accept the `Tooltip.Position` in here. Who knows if we'll need tooltips to the left or right someday.